src/Nio/AlsaEngine.cpp

```diff
@@ -221,9 +221,17 @@ void *AlsaEngine::MidiThread(void)
                 break;
 
             default:
-                if(true)
```

Comment:
So for MIDI, this will only work for ALSA, but not e.g. Jack?

Reply:
i would have thought that in jack it already worked. but i don't remember.